decidim-core/app/helpers/decidim/translations_helper.rb

```diff
@@ -11,8 +11,12 @@ module TranslationsHelper
     #
     # Returns a String with the translation.
     def translated_attribute(attribute)
+      return "" if attribute.nil?
```

Comment:
Is this needed? Is this method ever expected to be called with a nil argument? If not, I'd prefer to let things blow up.

Reply:
I guess `nil` can happen on optional attributes that are set without a default "" value. Maybe exploding is too defensive on a rendering context?

Reply:
You mean too offensive? Yeah, maybe. I would expect tests to fail when removing the line if it's needed, but that can be checked/ensured in a separate PR.